### PR TITLE
Bug fix spotify search confirmation alert

### PR DESCRIPTION
**Pull request by m4ndycheung**
On clicking the Submit a song button, the alert message popped up with undefined.

### CODE REVIEW

**Comment by m4ndycheung** [comment]
For: [Revert "Amended response message to include song name"], I hit commit too soon. SO reverted and tried again!

File: server/modules/apiCallToAddSongsToDB.js [hunk 1]
```diff
@@ -10,13 +10,16 @@ const apiCallToAddSongsToDB = function (req, res) {
 
   db.query(query, [artist_name, song_name, spotify_song_id])
     .then(() => {
-      res.status(201).send({ result: "Song added successfully, enjoy!" });
+      res.status(201).send({
+        result: "success",
+        message: `${song_name} by ${artist_name} added successfully!`,
```
Comment (ManSangSin) [inline server/modules/apiCallToAddSongsToDB.js:15]: Nice change! I like how the song name and artist is in the response message. It'll let users confirm which song is added

File: server/modules/apiCallToAddSongsToDB.js [hunk 2]
```diff
@@ -10,13 +10,16 @@ const apiCallToAddSongsToDB = function (req, res) {
...
+      });
     })
     .catch((err) => {
       console.log(err);
       res
         .status(400)
-        .send({ result: "failure", message: "Song could not be added" });
+        .send({ result: "failure", message: "Song could not be added." });
```
Comment (ManSangSin) [inline server/modules/apiCallToAddSongsToDB.js:22]: very punctual!

**Review by ManSangSin — APPROVED** [review]
🚀

**Pull request merged**